pkg/controller/migmigration/task.go

```diff
@@ -1362,6 +1379,66 @@ func (t *Task) hasImageStreams() (bool, error) {
 	return false, nil
 }
 
+// Get whether the associated plan has relevant artifacts using the analytics resource.
+func (t *Task) hasRelevantArtifacts() (bool, string, error) {
+	noMsg := ""
+	client, err := t.getDestinationClient()
```

Comment:
This does not seem appropriate, we should be using `t.Client` (controller client) rather than the destination client to interact with MigAnalytics, since the controller client will always provide access to Mig* resources.